config/local/gateway-service.yml

```diff
@@ -5,7 +5,9 @@ apiml:
         hostname: localhost
         ipAddress: 127.0.0.1
         port: 10010
+        centralRegistryUrls: https://localhost:10021/eureka/,https://localhost:10031/eureka/
```

Comment:
I want to mention that centralRegistryUrls has a different meaning than discoveryServiceUrls. It is mainly about HA.

discoveryServiceUrls contains all DS instances in the HA. It means, that the service could notify them but the heartbeat is sent just to one available instance. Basically, the second one and next values are used just in case the previous ones are not available. The value from the heartbeat is spread between other instances via peer calls.

The centralRegistryUrls defines a list of services that will be notified. In the case one of them will run in HA (for example central GW in HA) it is necessary to list all instances. It means there is redundant communication because we notify each instance instance of one and the peer communication between them will remain.

The correct (or let's say equivalent) communication should be based on a different configuration like:
```
centralRegistryUrls:
   apiml1:
       discoveryServiceUrls: https://localhost:10021/eureka/,https://localhost:10031/eureka/
   apiml2:
       discoveryServiceUrls: https://localhost:10041/eureka/
```

The Gateway should send a heartbeat to one of https://localhost:10021/eureka/,https://localhost:10031/eureka/ and to https://localhost:10041/eureka/. The HA instances should be grouped to simulate the same communication.

Reply:
thanks, it may be better to support HA here

Reply:
will be done in subsequent PR